src/OptionsPanel/OptionsList.tsx

```diff
@@ -96,8 +97,9 @@ export default function OptionsList({
                                             handleUpdateOptionValue={handleUpdateOptionValue(index)}
                                             handleUpdateOptionChecked={handleUpdateOptionChecked(index, multiple)}
                                             readOnly={readOnly}
-                                            disabled={
+                                            disableSoloCheckedOption={
```

Comment:
@glaubersilva thanks! one last tweak here - let's keep this `disabled` prop on `OptionsItem` as options should be able to be  disabled.

The logic here can remain as we're first checking `disableSoloCheckedOption` from the parent component.


```suggestion
                                            disabled={
```

Reply:
Resolved in f823f427045a5ca9ec6e5891c436dc4ac41df9f2